docs/docs/maintaining-a-plugin.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Maintaining a Plugin
+---
+
+This is a stub article meant to be filled with tips on how to maintain a Gatsby plugin once you've published it as an npm package.
+
+Topics to be covered:
+
+- yarn workspaces can solve yarn link inconsistencies
```

Comment:
@jlengstorf or @DSchau want to add any more details here?

Reply:
I would think this could have some technical bits and some social OSS bits, including:

## Technical
- Following [semver](https://semver.org/) for releasing the plugin
- Periodically putting out releases with bug fixes and improvements
- Testing for compatibility

## Social
- Recruiting help to maintain a plugin so it doesn't die on the vine
- Pair programming with Gatsby to keep things moving?